composer.json

```diff
@@ -49,6 +49,7 @@
         "vimeo/psalm": "^3.11"
     },
     "suggest": {
+        "doctrine/sql-formatter": "SQL statement formatter. Can be used for syntax highlighting too,",
```

Comment:
I don't see it being used by the library. What does the suggestion mean?

Reply:
For discoverability purposes

Reply:
> Suggested packages that can enhance or work well with this package.

I'm not really sold. The formatted doesn't enhance the work of DBAL and can work well with formatting any SQL, not only the one generated by DBAL.

An example of appropriate suggestions for DBAL would be `ext-mysql`, `ext-oci8`, etc. that the library can use but they are not required.

Reply:
Sure. But a "suggest" doesn't mean it only works with something, does it?

Maybe it should be wired in? ;)

The problem as per https://github.com/doctrine/dbal/issues/4003 is it wasn't obvious at all that you had a formatter available.

Reply:
Also, by the definition of "work well with this package"... this seems to meet that

Reply:
It doesn't work with the package. It works with SQL. Does it mean that any package that generates SQL should suggest this formatter? Since the formatting is about developer experience, should DBAL suggest all development tools like Xdebug, etc.?

This looks totally unnecessary to me.